packages/console/src/App.module.scss

```diff
@@ -1,4 +1,9 @@
 .skeleton {
+
```

Comment:
Consider renaming `skeleton` to `app`? For skeleton may be more suitable for those `skeleton screens` that show up when loading page contents

Reply:
agreed